canonical-web-and-design/jaas-dashboard
# Remove nanoid as a dependency

## Done

We no longer need this to support react-components as ContextualMenu, where this is used, was decoupled from Search And Filter.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Verify no errors
